transformer/diff.go

```diff
@@ -2,13 +2,15 @@ package transformer
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 )
 
 type DiffTransformer struct {
-	rawDiff   string
-	fileDiffs []string
-	segments  []string
+	rawDiff         string
```

Comment:
Suggestion: Consider importing "regexp" package only where it is used rather than at the top-level.